travis/pull_request_setup.bash

```diff
@@ -0,0 +1 @@
+#!/bin/bash
```

Comment:
I'm not sure if there's any set up required for pull requests (if there is then it'll be added there). If nothing needs to be done then it can be removed.
@cdeszaq ?